templates/web/common/includes/display_field.tt.html

```diff
@@ -1,8 +1,20 @@
 <!-- start templates/[% template.name %] -->
+<!-- Special handling to show a Hunger Game promo -->
+[% IF FIELD == 'BRAND' %]
+<p id="field_[% field %]">
+    <span class="field">[% name %][% sep %]: </span>
+    <span class="field_value" id="field_[% field %]_value">[% value %]</span>
+    <sup>[<a href="https://hunger.openfoodfacts.org/questions?value_tag=[% field %]&amp;type=brand" title="Help with this brand on Hunger Games">Help with this brand on Hunger Games</a>]</sup>
```

Comment:
It's not as simple, [% value %] contains a comma separated list of links, so we can't construct an hunger game URL like that.

e.g. if we have 2 brands, <span class="field_value" id="field_[% field %]_value">[% value %]</span> will generate this:

<span class="field_value" id="field_brands_value"><a itemprop="brand" href="/marque/activia">Activia</a>, <a itemprop="brand" href="/marque/danone">Danone</a></span>